api/src/main/java/org/openmrs/api/OrderService.java

```diff
@@ -834,6 +834,20 @@ public Order discontinueOrder(Order orderToDiscontinue, String reasonNonCoded, D
 	@Authorized({ PrivilegeConstants.EDIT_ORDERS, PrivilegeConstants.ADD_ORDERS })
 	public OrderGroup saveOrderGroup(OrderGroup orderGroup) throws APIException;
 
+	/**
+	 * Saves an order group with a specific order context
+	 *
+	 * @param orderGroup the order group to be saved
+	 * @param orderContext the order context data transfer object containing care setting and 
+	 * the order type to save with the order group
+	 * @return the order group that was saved with the specified order context data
+	 * @since 2.5.0
+	 * @should throw an exception if no order context is provided
```

Comment:
This `@should` annotation is not correct.  We allow order context to be null.  This should be removed.